SafeguardSessions/modules/schema/SchemaUtils.pqm

```diff
@@ -184,14 +207,19 @@ let
             withType = Value.ReplaceType(enforcedTypes, SchemaToTableType(schema))
         in
             withType,
-    SchemaUtils.ApplySchema = (response as record, schema as table) =>
+    SchemaUtils.ApplySchema = (response as record, schema as table, optional mocks as nullable record) =>
         let
             transformedResponse = SchemaUtils.GetSessionItemsResponse(response),
             flattenedResponse = SchemaUtils.Flatten(transformedResponse, {"vault"}),
             withSchema = SchemaUtils.GetResponseWithSchema(flattenedResponse, schema),
-            withMeta = Value.ReplaceMetadata(withSchema, Value.Metadata(response))

```

Comment:
1. Try removes the meta. Replacing meta should be the last step (for safety).

2. withSchema should mocked
if mocked = mocks[ApplySchema] else SchemaUtils.GetResponseWithSchema
The the processed part will be much shorted.

Reply:
ACK.